package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "ChronoFactorem",
+  "version": "1.0.0",
+  "description": "Timetable maker-helper",
+  "homepage": "https://github.com/Dryft-bits/ChronoFactorem#readme",
+  "author": "Dryft-bits",
+  "main": "server.jsx",
+  "dependencies": {
+    "bcryptjs": "^2.4.3",
+    "body-parser": "^1.18.3",
+    "config": "^3.2.5",
+    "cookie-parser": "^1.4.3",
+    "debug": "^4.1.0",
+    "dotenv": "6.2.0",
+    "express": "^4.17.1",
+    "express-validator": "^6.4.0",
+    "jsonwebtoken": "^8.5.1",
+    "mongoose": "^5.8.11",
+    "node-sass": "^4.5.3",
+    "node-sass-middleware": "^0.11.0",
+    "react": "^16.6.3",
+    "react-dom": "^16.6.3"
```

Comment:
Why are these in both client and here?

Reply:
`npx create-react-app client` put that there by default. Not sure whether removing it would cause problems.

Reply:
Not needed. Will remove with next commit.